### PR TITLE
Test: increase common module unit test coverage

### PR DESCRIPTION
**Pull request by l81893521**
- [ ] I have registered the PR [changes](../changes).

### Ⅰ. Describe what this PR did
The coverage please see https://app.codecov.io/gh/apache/incubator-seata/pull/6157/tree

### Ⅱ. Does this pull request fix one issue?
#6158

### CODE REVIEW

File: common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * limitations under the License.
+ */
+package io.seata.common.exception;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @description:
+ * @author: zhangjiawei
```
Comment (lightClouds917) [inline common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java]: you need remove @author: zhangjiawei;

**Review by lightClouds917 — CHANGES_REQUESTED** [review]

File: common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * @date: 2023/12/22
```
Comment (xingfudeshi) [inline common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java]: Please remove @date and @description

File: common/src/test/java/io/seata/common/exception/JsonParseExceptionTest.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ */
+package io.seata.common.exception;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @description:
+ * @author: zhangjiawei
+ * @date: 2023/12/21
```
Comment (xingfudeshi) [inline common/src/test/java/io/seata/common/exception/JsonParseExceptionTest.java]: Please remove @date  @description @author

File: common/src/test/java/io/seata/common/exception/RedisExceptionTest.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+ */
+package io.seata.common.exception;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @description:
+ * @author: zhangjiawei
+ * @date: 2023/12/21
```
Comment (xingfudeshi) [inline common/src/test/java/io/seata/common/exception/RedisExceptionTest.java]: Please remove @date @description @author

File: common/src/test/java/io/seata/common/exception/RetryableExceptionTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+ */
+package io.seata.common.exception;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @description:
+ * @author: zhangjiawei
+ * @date: 2023/12/21
```
Comment (xingfudeshi) [inline common/src/test/java/io/seata/common/exception/RetryableExceptionTest.java]: Please remove @date @description @author

**Review by xingfudeshi — DISMISSED** [review]
Please remove @date @description @author .
I'm ok with other changes.

File: common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java (hunk 1 above)
Comment (l81893521) [reply common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java]: fixed

File: common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java (hunk 2 above)
Comment (l81893521) [reply common/src/test/java/io/seata/common/exception/AuthenticationFailedExceptionTest.java]: fixed

File: common/src/test/java/io/seata/common/exception/JsonParseExceptionTest.java (hunk 3 above)
Comment (l81893521) [reply common/src/test/java/io/seata/common/exception/JsonParseExceptionTest.java]: fixed

File: common/src/test/java/io/seata/common/exception/RedisExceptionTest.java (hunk 4 above)
Comment (l81893521) [reply common/src/test/java/io/seata/common/exception/RedisExceptionTest.java]: fixed

File: common/src/test/java/io/seata/common/exception/RetryableExceptionTest.java (hunk 5 above)
Comment (l81893521) [reply common/src/test/java/io/seata/common/exception/RetryableExceptionTest.java]: fixed

**Review by xingfudeshi — APPROVED** [review]
LGTM

**Pull request merged**